www-servers/pound/pound-3.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_P=${P/p/P}
+DESCRIPTION="A http/https reverse-proxy and load-balancer"
+HOMEPAGE="https://www.apsis.ch/pound.html"
+SRC_URI="http://www.apsis.ch/pound/${MY_P}.tgz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~x86"
+
+DEPEND="
+	dev-libs/libpcre
+	dev-libs/libyaml
+	dev-libs/nanomsg
+	dev-libs/openssl
+	net-libs/mbedtls
+"
+
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+
+src_compile() {
+	rm -r GPL.txt || die
+	cmake_src_compile
+}
+
+src_install() {
+	dodir /usr/sbin
+	cp "${BUILD_DIR}"/pound "${ED}"/usr/sbin/ || die
```

Comment:
`dosbin` / `newsbin`?
https://devmanual.gentoo.org/function-reference/install-functions/

Reply:
Also I want to point out that pound-2.7f installs two binaries into /usr/sbin, you're only installing one.

Reply:
On 3.0 there is only one binary file